WPFPractice2019.12.21/WPFTask2.0/MainWindow.xaml

```diff
@@ -0,0 +1,49 @@
+﻿<Window x:Class="WPFTask2._0.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:sys="clr-namespace:System;assembly=mscorlib"
+        xmlns:local="clr-namespace:WPFTask2._0"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="150" Width="200">
+    <Window.Resources>
+        <sys:String x:Key="FirstName">Egor</sys:String>
+        <sys:String x:Key="LastName">Putik</sys:String>
+        <sys:Boolean x:Key="MasterDegree">True</sys:Boolean>
+    </Window.Resources>
+    <Window.DataContext>
+        <local:Handler/>
+    </Window.DataContext>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="15*"/>
+            <ColumnDefinition Width="50*"/>
+            <ColumnDefinition Width="50*"/>
+            <ColumnDefinition Width="77*"/>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition/>
+            <RowDefinition/>
+            <RowDefinition/>
+        </Grid.RowDefinitions>
+        <Label Content="First name" Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2" VerticalAlignment="Center" Height="26" Margin="0,7"/>
+        <Label Content="Last name" Grid.Row="1" Grid.Column="0" Grid.ColumnSpan="2" VerticalAlignment="Center" Height="26" Margin="0,7"/>
+        <!--Binding with primitives or with custom class (check MainWindow.cs)-->
+        <TextBox Grid.Row="0" Grid.Column="2" Grid.ColumnSpan="2" Margin="5" Text="{Binding Path=myStudent.FirstName}"/>
+        <TextBox Grid.Row="1" Grid.Column="2" Grid.ColumnSpan="2" Margin="5" Text="{Binding Path=myStudent.LastName}"/>
+        <CheckBox Grid.Row="2" Grid.Column="1" Grid.ColumnSpan="2" Content="Master degree" IsChecked="{Binding Path=myStudent.MasterDegree}" VerticalAlignment="Center" HorizontalAlignment="Right" Height="15" Margin="0,12" Width="97"/>
+
+        <!--StaticResource with primitives-->
+        <!--<TextBox Grid.Row="0" Grid.Column="2" Grid.ColumnSpan="2" Margin="5" Text="{StaticResource FirstName}"/>
+        <TextBox Grid.Row="1" Grid.Column="2" Grid.ColumnSpan="2" Margin="5" Text="{StaticResource LastName}"/>
+        <CheckBox Grid.Row="2" Grid.Column="1" Grid.ColumnSpan="2" Content="Master degree" IsChecked="{StaticResource MasterDegree}" VerticalAlignment="Center" HorizontalAlignment="Right" Height="15" Margin="0,12" Width="97"/>-->
+
+        <!--StaticResource with custom class-->
```

Comment:
3. Static resource with custom class - do not see the resource with custom class.